cel/options.go

```diff
@@ -328,6 +329,18 @@ func registerFiles(reg ref.TypeRegistry, files *protoregistry.Files) error {
 	return err
 }
 
+// Locale configures the Env with the given locale tag.
```

Comment:
Document what happens if Locale is not explicitly set?

Reply:
Right, I've added more documentation to `Locale` that explains default behavior.